CHANGELOG.md

```diff
@@ -75,6 +75,7 @@ By @Vecvec in [#7913](https://github.com/gfx-rs/wgpu/pull/7913).
 - The function you pass to `Device::on_uncaptured_error()` must now implement `Sync` in addition to `Send`, and be wrapped in `Arc` instead of `Box`.
   In exchange for this, it is no longer possible for calling `wgpu` functions while in that callback to cause a deadlock (not that we encourage you to actually do that).
   By @kpreid in [#8011](https://github.com/gfx-rs/wgpu/pull/8011).
+- `wgpu` now requires that the requested device limits satisfy `min_subgroup_size <= max_subgroup_size`. By @andyleiserson in [#8085](https://github.com/gfx-rs/wgpu/pull/8085).
```

Comment:
question: Does this not also apply to `wgpu-core`? 🤔

Reply:
I'm not sure if it was on a different PR or I hallucinated it, but when I was revising I thought the suggestion was that specifying `wgpu` here was redundant and not a good use of precious changelog verbiage. Based on that I reworded it to "The limits requested for a device must now satisfy...".

To answer the question that you actually asked here, it applies to either `wgpu` or `wgpu-core`. I think the new text adequately reflects that.